src/typecheck/mod.rs

```diff
@@ -1066,6 +1091,16 @@ pub fn apparent_type(t: &Term, envs: Option<&Envs>) -> ApparentType {
             .and_then(|envs| envs.get(id))
             .map(ApparentType::FromEnv)
             .unwrap_or(ApparentType::Approximated(Types(AbsType::Dyn()))),
+        Term::ResolvedImport(f) => {
+            if let Some(r) = resolver {
+                let t = r.get(*f).expect(
+                    "Internal error: resolved import not found ({:?}) during typechecking.",
```

Comment:
`expect` takes a `&str` (and you haven't passed any parameter). So this will always print `{:?}` literally in the message. You either need to use `format!` or decide not to care about showing the specific import and drop the interpolation altogether.

Reply:
Oops indeed, I just copied the line 970 (bottom of the `type_check_` function) which has exactly the same problem apparently, gonna push a fix for that too right after